cypress/integration/requests/requests-sort.spec.js

```diff
@@ -0,0 +1,80 @@
+import testType from '../../support/dictionary/testTypes';
+import TopMenu from '../../support/fragments/topMenu';
+import Requests from '../../support/fragments/requests/requests';
+import EditRequest from '../../support/fragments/requests/edit-request';
+import { MultiColumnListHeader } from '../../../interactors';
+
+describe('ui-requests: Sort requests', () => {
+  const usersData = [];
+  const requestsData = [];
+  const cancellationReasons = [];
+
+  before(() => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.getToken('diku_admin', 'admin');
+  });
+
+  beforeEach(() => {
+    Requests.requestTypes.forEach((requestType) => {
+      const itemStatus = requestType === 'Page' ? 'Available' : 'Checked out';
+      Requests.createRequestApi(itemStatus, requestType).then(({
+        createdUser,
+        createdRequest,
+        cancellationReasonId
+      }) => {
+        usersData.push(createdUser);
+        requestsData.push(createdRequest);
+        cancellationReasons.push(cancellationReasonId);
+      });
+    });
+  });
+
+  afterEach(() => {
+    Requests.requestTypes.forEach((_, i) => {
+      EditRequest.updateRequestApi({
+        ...requestsData[i],
+        status: 'Closed - Cancelled',
+        cancelledByUserId: requestsData[i].requesterId,
+        cancellationReasonId: cancellationReasons[i],
+        cancelledDate: new Date().toISOString(),
+      });
+      cy.deleteUser(usersData[i].id);
+    });
+  });
+
+  it('should check all the request types and validate sorting', { tags: [testType.smoke] }, () => {
```

Comment:
can't find implementation of check "Check that the requests are sorted by Request date (oldest at top) by default". Is it implemented?